collectors/proc.plugin/multi_metadata.yaml

```diff
@@ -4632,7 +4734,12 @@ modules:
     troubleshooting:
       problems:
         list: []
-    alerts: []
+    alerts:
+      - name: linux_power_supply_capacity
+        link: 
+          https://github.com/netdata/netdata/blob/master/health/health.d/linux_power_supply.conf
```

Comment:
@Ancairon we have another alert here

Reply:
these warnings are not entirely true, the yaml validates fine, will have to see if I can prevent this through ruamel.yaml.